server/controllers/UsersController.js

```diff
@@ -98,6 +100,128 @@ class UsersController {
         }
       }));
   }
+
+  /**
+ * @description - Route handler to send password reset link
+ * @param {object} req - request object
+ * @param {object} res - response object
+ * @returns {object} - returns object
+ */
+  static forgotPassword(req, res) {
+    const { email } = req.body;
+    const lifeSpan = '1h'; // token expires after 1 hour
+
+    User.findOne({
+      where: {
+        email
+      }
+    })
+      .then((userFound) => {
+        if (!userFound) {
+          return res.status(404).json({
+            errors: {
+              message: ['No account with that email address exists']
+            }
+          });
+        }
+
+        const userId = userFound.id;
+        const token = createToken(userId, lifeSpan);
+        const forgotUrl = `${req.protocol}://${req.host}/password_reset`
+        + `/${token}`;
```

Comment:
This `forgotUrl` will be `/password_reset/token`. This is different from the actual route you created which is `/api/v1/users/reset/:token`. So when the user clicks the link, it won't trigger the route. 

Everything else looks good, so good job on that.

Reply:
Yeah, I can see now, thanks for spotting this error.